augly/video/functional.py

```diff
@@ -706,8 +706,7 @@ def hflip(
     """
     func_kwargs = helpers.get_func_kwargs(metadata, locals(), video_path)
 
-    hflip_aug = af.VideoAugmenterByHFlip()
-    vdutils.apply_ffmpeg_augmenter(hflip_aug, video_path, output_path)
+    af.VideoAugmenterByHFlip.add_augmenter(video_path, output_path)
```

Comment:
```suggestion
    af.VideoAugmenterByHFlip().add_augmenter(video_path, output_path)
```

This is causing your error :) because you weren't instantiating the `VideoAugmenterByHFlip` object, you were just calling the function `add_augmenter` from that class, but it expects the `self` arg which you weren't passing (when you call a class method on an object then `self` gets passed implicitly). So it was interpreting `video_path` as `self` & `output_path` as `video_path` -- but it was complaining that `output_path` didn't exist yet (which it would need to if it was the src video path). This should fix it!